store/postgres/src/relational_queries.rs

```diff
@@ -586,6 +558,49 @@ impl EntityData {
     }
 }
 
+#[derive(QueryableByName, Clone, Debug, Default, Eq)]
+pub struct EntityDataExt {
+    #[diesel(sql_type = Text)]
+    pub entity: String,
+    #[diesel(sql_type = Jsonb)]
+    pub data: serde_json::Value,
+    #[diesel(sql_type = Integer)]
+    pub block_number: i32,
+    #[diesel(sql_type = Text)]
+    pub id: String,
+    #[diesel(sql_type = BigInt)]
+    pub vid: i64,
+}
+
+impl Ord for EntityDataExt {
+    fn cmp(&self, other: &Self) -> Ordering {
+        let ord = self.block_number.cmp(&other.block_number);
+        if ord != Ordering::Equal {
+            ord
+        } else {
+            // TODO: check if this and the next cmp for string match postgress C localle
```

Comment:
Rust string comparison is basically the same as memcmp from libc (byte per byte binary comparison). The LC_COLLATE=C in Postgress should be the same.